openml/datasets/dataset.py

```diff
@@ -1070,6 +1070,7 @@ def _parse_features_xml(features_xml_string: str) -> dict[int, OpenMLDataFeature
             xmlfeature["oml:data_type"],
             xmlfeature.get("oml:nominal_value"),
             int(nr_missing),
+            xmlfeature.get("oml:ontology"),
```

Comment:
Same as above, requires a default or assumes None as default

Reply:
can you elaborate what you mean? This is the usage of the constructor, right?